compiler/src/dmd/lexer.d

```diff
@@ -121,6 +125,8 @@ class Lexer
         ErrorSink errorSink,
         const CompileEnv* compileEnv) scope
     {
+        import dmd.globals;
```

Comment:
I went to considerable effort to remove all globals from lexer! Instead of importing it, add the needed configuration to CompileEnv.